src/dask_awkward/lib/structure.py

```diff
@@ -691,6 +692,15 @@ def ones_like(
     )
 
 
+@borrow_docstring(ak.to_list)
+def to_list(array: Array) -> list:
+    """Return list/dict version of the data
+
+    Unlike most functions, this one requires a compute() of the data.
+    """
+    return array.compute().to_list()
```

Comment:
@jpivarski I wonder, should this be lazy and return a bag of lists?

Reply:
We chatted in the meeting and thought not - this is only likely to be used in testing for small data.